cmd/kubeadm/app/cmd/reset.go

```diff
@@ -82,6 +87,11 @@ func NewCmdReset(out io.Writer) *cobra.Command {
 		"The path to the CRI socket to use with crictl when cleaning up containers.",
 	)
 
+	cmd.PersistentFlags().BoolVar(
+		&userConfirmedReset, "confirm", false,
```

Comment:
What about --force or --silent instead?

Reply:
Heh, I had force initially. I don't have a strong opinion here, so happy to make the change if we think that's better.